script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2B.txt

```diff
@@ -101,9 +101,9 @@ And then, I was attacked by a passing serial killer, which was quite troublesome
 She claps her hands together, smiling at me.
 It seems that she's taking a dig at said passing serial killer. How uncomfortable.
 <Page31>
-"...I see. I think I'm starting to get the picture.
-So a monster that preys on humans built a nest in this city, and Arcueid is here to exterminate it.
-But as she was looking for its hiding place, that's when she was, well, attacked by me. And now because she's weak, she needs to hide until she recovers... did I get that right?"
+"...I see. I think I'm starting to get the picture.//to be rechecked
+So a monster that preys on humans built a nest in this city, and you're here to exterminate it.
+But as you're looking for its hiding place, that's when you're, well, attacked by me. And now because you're weak, you need to hide until you recover... did I get that right?"
```

Comment:
But **while you were** looking for its hiding place, that's when **you were**, well, attacked by me. And because you're weak **now**, you need to hide until you recover